src/warc_types.rs

```diff
@@ -180,11 +181,190 @@ impl<R: BufRead> Iterator for WarcReader<R> {
             version: version_ref.to_owned(),
             headers: headers_ref
                 .into_iter()
-                .map(|(token, value)| (token.to_lowercase().to_owned(), value.to_owned()))
+                .map(|(token, value)| (token.into(), value.to_owned()))
```

Comment:
I think we should keep normalizing all header tokens. Consumers won't be able to deterministically know or guess the casing used in an arbitrary warc file.

Reply:
I just realized my PR description was wrong: the cast from a string _will_ convert unknown tokens to lower case. See line 76 of `header.rs`.

That means I don't have to do anything here, correct? I will update the PR description.